src/CBT.Microsoft.CSharp.targets/Module/Build.props

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
+  <PropertyGroup Condition=" '$(MSBuildProjectExtension)' == '.csproj' Or '$(ForceImportCBTMicrosoftCSharpTargets)' == 'true' ">
```

Comment:
we can try based off the project extension given the override.  My concern is many projects will just import the csharp targets even though they may not be a .csproj.  Of course there are more correct ways to do what they want and providing the property they need to set manually should be sufficient if they want this functionality especially since in this scenario they are doing custom things.